### PR TITLE
Update for alp auto-installation

### PR DESCRIPTION
**Pull request by lemon-suse**
We need update for alp auto-installation following comments on PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16927

- Related ticket: https://progress.opensuse.org/issues/127349
- Needles: tags: agama-main-page, agama-installing, welcome-to, merged in OSD.
- Verification run: https://openqa.opensuse.org/tests/3241681#  x86_64
                                https://openqa.opensuse.org/tests/3241920#  aarch64

### CODE REVIEW

File: schedule/yam/agama_auto.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: agama_auto
+description: >
+  Prepare url for agama.auto boot parameter, boot, perform auto-installation for autoinstallation with agama.
```
Comment (jknphy) [inline schedule/yam/agama_auto.yaml:4]: we can remove "for autoinstallation" in this sentence.

File: tests/installation/bootloader_start.pm [hunk 2]
```diff
@@ -63,6 +63,11 @@ sub run {
         # As normally we also need `bootloader` for this scenario
         return if get_var('MEDIACHECK');
     }
+    # Prepare for agama.auto
```
Comment (jknphy) [inline tests/installation/bootloader_start.pm:66]: this code should be in `lib/bootloader_setup.pm`

**Comment by jknphy** [comment]
regarding needles:
- better not include icon in needle, just the text as the icon might change more frequently that the font or the aliment between them: https://openqa.opensuse.org/tests/3241583#step/auto/2
- To make the needle useful for other products, we could just check Install button instead of specific product:
https://openqa.opensuse.org/tests/3241583#step/auto/1

**Comment by jknphy** [comment]
For VR you need to test in both architectures.

File: schedule/yam/agama_auto.yaml (hunk 1 above)
Comment (lemon-suse) [reply schedule/yam/agama_auto.yaml]: Updated.

File: tests/installation/bootloader_start.pm (hunk 2 above)
Comment (lemon-suse) [reply tests/installation/bootloader_start.pm]: Updated. VR is running.

File: lib/bootloader_setup.pm [hunk 3]
```diff
@@ -505,6 +505,10 @@ sub bootmenu_default_params {
         }
 
     }
+    if (get_var('AGAMA_AUTO')) {
+        my $path = data_url(get_var('AGAMA_AUTO'));
+        set_var('EXTRABOOTPARAMS', "agama.auto=\"$path\"");
```
Comment (jknphy) [inline lib/bootloader_setup.pm:510]: I guess if EXTRABOOTPARAMS is already set, we will overwrite or perhaps is running before, worth a comment about it if this functionality is limited at the moment, wdyt?

**Comment by jknphy** [comment]
> For VR you need to test in both architectures.

remember that you don't need to use the Playwright live-cd, but the [ALP one](https://build.opensuse.org/package/binaries/YaST:Head:Agama/agama-live:ALP/images)

File: lib/bootloader_setup.pm (hunk 3 above)
Comment (jknphy) [reply lib/bootloader_setup.pm:510]: seems fine :525 handle those, so should be ok.

**Comment by lemon-suse** [comment]
VRs updated, both for x86_64 and aarch64.

**Pull request merged**